### PR TITLE
Bug/can not click on projects options if hide on mouse leave is active

### PR DESCRIPTION
**Pull request by DavidDamke**

### CODE REVIEW

File: src/main/java/de/doubleslash/keeptime/view/ViewController.java [hunk 1]
```diff
@@ -150,6 +150,7 @@ private class Delta {
    private final BooleanProperty mouseHoveringProperty = new SimpleBooleanProperty(false);
    public static final LongProperty activeWorkSecondsProperty = new SimpleLongProperty(0);
    public static final ObjectProperty<Color> fontColorProperty = new SimpleObjectProperty<>();
+   boolean contexMenuOpenBoolean = false;
```
Comment (Death111) [inline src/main/java/de/doubleslash/keeptime/view/ViewController.java:153]: can you add appropriate modifiers to the property?

**Review by Death111 — CHANGES_REQUESTED** [review]

File: src/main/java/de/doubleslash/keeptime/view/ViewController.java (hunk 1 above)
Comment (DavidDamke) [reply src/main/java/de/doubleslash/keeptime/view/ViewController.java:153]: Changed Modifiers to private static boolean contextMenuOpenBoolean

File: src/main/java/de/doubleslash/keeptime/view/ViewController.java [hunk 2]
```diff
@@ -150,6 +150,7 @@ private class Delta {
...
```
[the remark this answers is not in the document]
Comment (DavidDamke) [reply src/main/java/de/doubleslash/keeptime/view/ViewController.java:154]: Changed name to "App" Maybe not the best name but I cant think of a better one right now.

File: src/main/java/de/doubleslash/keeptime/view/ViewController.java [hunk 3]
```diff
@@ -268,8 +269,19 @@ private void initialize() {
 
       pane.setOnMouseEntered(a -> mouseHoveringProperty.set(true));
 
-      pane.setOnMouseExited(a -> mouseHoveringProperty.set(false));
+      projectsVBox.setOnContextMenuRequested(c -> { // Is needed because the Context menu loses Focus otherwise
+         mouseHoveringProperty.set(true);
+         LOG.info("Options selected");
```
[the remark this answers is not in the document]
Comment (DavidDamke) [reply src/main/java/de/doubleslash/keeptime/view/ViewController.java]: Changed it to trace

**Review by Death111 — APPROVED** [review]

**Pull request merged**